setup.cfg

```
@@ -35,7 +33,7 @@ classifiers =
 
 [options]
 zip_safe = False
-python_requires = >=2.7, !=3.0, !=3.1, !=3.2, !=3.3, !=3.4, !=3.5, !=3.6
+python_requires = >=3.7,!=3.10
```

```suggestion
python_requires = >=3.7
```

Never put an upper cap here. It doesn't work.

---

Not really related (excluding a version you never have supported in `python_requires` doesn't work, it's about back-solving for older compatible versions of your library), but do you know why Python 3.10 doesn't work?

---

I have not tried 3.10. You know that we maintain pyjet in low priority, so never tried. I think what is now important is to provide a new release (old one is really old) and make that effectively the "legacy release". Then direct people to fastjet.
We can add 3.10 if that works, of course 👍.